test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -0,0 +1,147 @@
+Describe "Basic Send-MailMessage tests" -Tags CI {
```

Comment:
Currently this test runs only on Linux. So long as the sender/receiver of the email is the currently logged-in user, running as root is not required.

I've been looking into ways to make tests more cross-platform, using a suggestion from @iSazonov, but it's going to take some effort.

My personal feeling is that I'd like to create a separate issue for testing `Send-MailMessage` and have this PR be taken as-is, but I don't know if that works with testing policies.